lambdas/mail_save.js

```diff
@@ -37,6 +40,24 @@ const handler = async (req, res) => {
               statuses['created'],
               'La suscripción se ha realizado correctamente'
             );
+
+            fs.readFile(path.resolve(__dirname, '../templates/mail_registration.html'), 'utf8', function(err, html) {
+              if (err) {
+                console.error(err);
```

Comment:
Since we already have the Analytics account working, we should probably implement [measurement protocol](https://developers.google.com/analytics/devguides/collection/protocol/v1/devguide) to track the errors somewhere.

I can help you with this, I've already done this on a express server.

Reply:
I disagree. Analytics is a tracking tool for user behaviour, not errors. If we're gonna add something similar, let's go with Sentry, New Relic or LogEntries or something more suited to it.

Reply:
Agree, I went with the free one :P. I would rather implement Sentry that measurement protocol (it's A LOT easier). Can we add it to the budget? Or should we try the free version?

Reply:
this feels like something that should be done in a separate PR, because it merits further discussion. I'll make an issue in trello